PowerShellEditorServices.build.ps1

```diff
@@ -129,6 +129,10 @@ task TestPowerShellApi -If { !$script:IsUnix } {
 task Build {
     exec { & $script:dotnetExe build -c $Configuration .\src\PowerShellEditorServices.Host\PowerShellEditorServices.Host.csproj $script:TargetFrameworksParam }
     exec { & $script:dotnetExe build -c $Configuration .\src\PowerShellEditorServices.VSCode\PowerShellEditorServices.VSCode.csproj $script:TargetFrameworksParam }
+    exec { & $script:dotnetExe publish -c $Configuration .\src\PowerShellEditorServices\PowerShellEditorServices.csproj -f netstandard1.6 }
```

Comment:
Just curious - why are you are calling `dotnet publish`? Does a normal build not have what you're looking for?

Reply:
Yeah, dependencies aren't copied on `build` when you target standard or core.  `publish` is supposed to the the final product with everything needed to run including packages, runtimes and framework assemblies.  Most of that is already provided by being hosted in PowerShell though, so I just pick out what's needed.